src/l10n.json

```diff
@@ -88,6 +88,7 @@
     "general.signIn": "Sign in",
     "general.startOver": "Start over",
     "general.statistics": "Statistics",
+    "genral.story": "Create a Story",
```

Comment:
Is this string necessary? I don't see it used in the code. (Also, "general is misspelled :) )

Reply:
That string was not necessary. I removed it.